### ISSUE TITLE
Interplay with Org-Roam not Working

### ISSUE THREAD

**Issue by SvenBre** [org-noter/org-noter#43]
### Description
I saw your video about using org-noter together with org-roam on Youtube. Does not seem to work for me. How did you do it? Here is the content of an org file which is supposed to be an org-noter file and an org-roam file at the same time:

`:PROPERTIES:
 :ID:       2224ab6f-883f-4a8d-a194-519cc9c38135
 :ROAM_REFS: somecitekey
 :END:
 #+title: Some Title
 #+year: 2004
 #+filetags: Scholar

- tags :: [[id:a93d0a02-55e8-431f-a936-a733cb38a078][Some tag]]

* Some author, Some title
  :PROPERTIES:
  :NOTER_DOCUMENT: ../../secondary/somefilename.pdf
  :END:
** Some note
    :PROPERTIES:
    :ID:       0a3a1902-92dc-4f9b-bb3a-40004d4d09cf
    :NOTER_PAGE: 5
    :END:

Some text.
`

The NOTER_PAGE property obviously prevents org-roam from recognizing the note's ID. Once this property line is deleted, the note "Some note" reappears in org-roam's database under its ID. Any idea?

**Comment by dmitrym0** [comment]
Hey @SvenBre, I should've been cleared in the video -- this is still very much a work in progress. It's tracked in #39.

**Comment by SvenBre** [comment]
Thanks. I saw the pull request (afterwards). Not working for me (symbol's value void as variable "org-roam-capture-templates"), why ever. Please tell me if I can be of help as a tester.

**Comment by dmitrym0** [comment]
@SvenBre org-roam support hasn't been merged yet, it lives in this pr still: https://github.com/org-noter/org-noter/pull/39. What are you testing?

I tried your org file with our "test" setup (see [config here](https://github.com/org-noter/org-noter-with-emacs-basic-test/blob/test/org-roam-integration/install-straight.el)) and it seems to work fine. Both nodes in your file show up.

My roam integration expects `(org-roam-db-autosync-mode)`, I wonder if you have it enabled?
<img width="857" alt="image" src="https://github.com/org-noter/org-noter/assets/707020/179ca30b-c853-4a70-8e0a-156be92fac68">

**Comment by SvenBre** [comment]
Hi, I have git cloned the feature/org-roam-integration branch and load it locally. Here is my config:

`
(use-package nov
  :ensure t)

(use-package djvu
  :ensure t)

(add-to-list 'load-path "~/elisp/org-noter")
(require 'org-noter)
(require 'org-noter-org-roam)

(setq org-noter-notes-search-path '("~/aktuell/library/RoamNotes/references"))
(setq org-noter-insert-selected-text-inside-note t)
(setq org-noter-highlight-selected-text t)
(setq org-noter-create-session-from-document-hook '(org-noter--create-session-from-document-file-supporting-org-roam))
(setq org-noter-always-create-frame nil)
(setq org-noter-property-note-location "n_page")

(global-set-key (kbd "<f11> n") 'org-noter)
`

Yes, org-roam-db-autosync-mode is enabled.

**Comment by dmitrym0** [comment]
@SvenBre is this the entirety of your config? It cannot be, because I don't see any references to org-roam. I suspect that like in many other cases the bug you're seeing is related to your custom config. I could not reproduce it with a "clean emacs" instance (see *full* config [here](https://github.com/org-noter/org-noter-with-emacs-basic-test/blob/test/org-roam-integration/install-straight.el)).

**Comment by SvenBre** [comment]
No, sure not. My config is almost 5000 lines. If it's working for you, something else in my setup must be the culprit. I'll try to find it by "brute force". But this will take some time. I'll keep you informed, especially if it turns out to be a principle package incompatibility. May be the org-gtd package that also breaks your wonderful org-hyperscheduler which I sadly can't use.
